libvips/foreign/magicksave.c

```diff
@@ -66,6 +66,7 @@
  * * @format: %gchararray, format to save as
  * * @optimize_gif_frames: %gboolean, apply GIF frames optimization
  * * @optimize_gif_transparency: %gboolean, apply GIF transparency optimization
+ * * @bitdepth: %gint, number of bits per pixel
```

Comment:
Could you add a short sentence to the docs below as well? Eg. 0 means no quantisation, don't try to set it to 16 to write a 16-bit image, etc.